package.json

```diff
@@ -114,6 +114,7 @@
   },
   "peerDependencies": {
     "typescript": "^5",
+    "@rhinestone/module-sdk": "^0.1.22",
```

Comment:
You'll need to update the readme and inform himanshu so that users know that they'll need to install rhinestone during installation when they run bun i ...